tools/docker/docker-compose-dev.yaml

```diff
@@ -217,6 +221,23 @@ services:
       - '../../:/app/src:z'
     restart: always
 
+  certs:
+    image: quay.io/fedora/fedora-minimal
+    volumes:
+      - ./postgres_ssl_config/certs:/etc/ssl/certs 
+    entrypoint: 
+      - sh
+      - -c
+      - |
+        sudo dnf -y install openssl
```

Comment:
I think this should be encapsulated in a script and used in https://github.com/ansible/eda-server/blob/main/tools/docker/docker-compose-dev-redis-tls.yaml

Reply:
great, I will move it to this.